rtsp_to_hls/lib/application.ex

```diff
@@ -5,7 +5,7 @@ defmodule Membrane.Demo.RtspToHls.Application do
   require Logger
   alias Membrane.Demo.RtspToHls.Pipeline
 
-  @rtsp_stream_url "rtsp://rtsp.membrane.work:554/testsrc.264"
+  @rtsp_stream_url "rtsp://root:dupa123@192.168.83.150:554/axis-media/media.amp?videocodec=h264&resolution=640x480"
```

Comment:
I am not sure if that stream's url is representative enough :D

Reply:
Out of the ones you sent me the dupa123 one is the only one that the demo is able to connect to and the original one still isn't working